### DISCUSSION TITLE
Macro recording does not register custom commands

### DISCUSSION THREAD

**Discussion by sourcevault** [NeoVintageous/NeoVintageous#988] [General]
https://github.com/NeoVintageous/NeoVintageous/assets/20347647/185a5524-30e5-479e-867c-045afd6d9871

So this is the plugin that I use :

https://github.com/randy3k/AlignTab

Unable to get NeoVintageous to properly call the custom script in the macro.

**Comment by gerardroche** [comment]
Thanks, I created an issue here  https://github.com/NeoVintageous/NeoVintageous/issues/989
